app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,41 @@
+class CommentsController < ApplicationController
+  load_and_authorize_resource except: %i[create]
+
+  def create
+    @resource = Issue.find(params[:issue_id])
+    authorize! :read, @resource
+
+    @comment = Comment.new(comment_params)
+    authorize! :create, @comment
```

Comment:
no need

Reply:
now only 'authorize! :read, \@resource' remains

Reply:
removed it completely